src/sql/parts/grid/views/editData/editData.component.ts

```diff
@@ -290,27 +300,20 @@ export class EditDataComponent extends GridParentComponent implements OnInit, On
 			});
 
 		if (this.currentCell.row !== row) {
-			// If we're currently adding a new row, only commit it if it has changes or the user is trying to add another new row
-			if (this.newRowVisible && this.currentCell.row === this.dataSet.dataRows.getLength() - 2 && !this.isNullRow(row) && this.currentEditCellValue === undefined) {
```

Comment:
Note : I'm removing this logic because I couldn't find a valid case for why we'd want to do this if we're changing cells we should always be committing that change to the data service I think